pyproject.toml

```diff
@@ -0,0 +1,32 @@
+[build-system]
+requires = [
+    "setuptools >= 65.3.0, < 66",
+    "setuptools_scm[toml] >= 7.0.5, < 8",
+    "wheel"
+]
+build-backend = "setuptools.build_meta"
+
+[tool.pylint]
+    [tool.pylint.'SIMILARITIES']
+    ignore-comments=['yes']
+    ignore-docstrings=['yes']
+    ignore-imports=['no']
+    min-similarity-lines=40
+
+    [tool.pylint.messages_control]
+    # disable wrong-import-order, ungrouped-imports because it conflicts with isort
+    disable = ["too-few-public-methods", "wrong-import-order", "ungrouped-imports"]
+    [tool.pylint.'DESIGN']
+    max-attributes=12
```

Comment:
Nor the extra indentation. I never seen this styles in other projects or other TOML files, I'll try to keep it with a more widely used format.

Reply:
this is updated as well.